tests/create.go

```diff
@@ -57,3 +61,39 @@ func testCreateRestartAlways(t *testing.T, sb integration.Sandbox) {
 	require.NoError(t, err, out)
 	builderName = strings.TrimSpace(out)
 }
+
+func testCreateRemoteContainer(t *testing.T, sb integration.Sandbox) {
+	if sb.Name() != "docker" {
+		t.Skip("skipping test for non-docker workers")
+	}
+
+	var ctnBuilderName, remoteBuilderName string
+	t.Cleanup(func() {
+		if ctnBuilderName != "" {
+			out, err := rmCmd(sb, withArgs(ctnBuilderName))
+			require.NoError(t, err, out)
+		}
+		if remoteBuilderName != "" {
+			out, err := rmCmd(sb, withArgs(remoteBuilderName))
+			require.NoError(t, err, out)
+		}
+	})
+
+	out, err := createCmd(sb, withArgs("--driver", "docker-container"))
+	require.NoError(t, err, out)
+	ctnBuilderName = strings.TrimSpace(out)
+
+	out, err = inspectCmd(sb, withArgs("--bootstrap", ctnBuilderName))
+	require.NoError(t, err, out)
+
+	cmd := dockerCmd(sb, withArgs("container", "inspect", fmt.Sprintf("%s%s0", driver.BuilderNamePrefix, ctnBuilderName)))
+	cmd.Stderr = os.Stderr
+	require.NoError(t, cmd.Run())
+
+	out, err = createCmd(sb, withArgs("--driver", "remote", fmt.Sprintf("docker-container://%s%s0", driver.BuilderNamePrefix, ctnBuilderName), "--bootstrap"))
+	require.NoError(t, err, out)
+	remoteBuilderName = strings.TrimSpace(out)
+
+	out, err = inspectCmd(sb, withArgs("--bootstrap", remoteBuilderName))
```

Comment:
This does not need bootstrap (actually the `create` in line 93 doesn't as well).

If we use a generated name for both builders then for extra safety check that the names are not the same and this line isn't just checking the first builder again.

Reply:
> then for extra safety check that the names are not the same

Don't think they can be the same, we have a check in store logic for this when generating name: https://github.com/docker/buildx/blob/4327ee73b18949e47eb1c865daa262af095cf974/store/util.go#L40-L54

Reply:
> This does not need bootstrap (actually the `create` in line 93 doesn't as well).

My bad forgot to remove this arg

Reply:
Yes, but the reason for the check is to detect the case if this code breaks.